src/parameter_declarations.ts

```diff
@@ -519,12 +518,11 @@ Parameter.declare({
 	forbiddenConjunctionParameters: [MetaDataFileParameter]
 });
 
-// TODO: PE-1534
-// Parameter.declare({
-// 	name: DataGqlTagsParameter,
-// 	aliases: ['--data-gql-tags'],
-// 	type: 'array',
-// 	description:
-// 		'(OPTIONAL) A mapping of custom metadata in the `"TAG_NAME" "TAG_VALUE"` format to be applied to the GQL Tags of all Data Transactions created. Must be an even number of string values to determine custom metadata. Can NOT be used in conjunction with --metadata-file',
-// 	forbiddenConjunctionParameters: [MetaDataFileParameter]
-// });
+Parameter.declare({
+	name: DataGqlTagsParameter,
+	aliases: ['--data-gql-tags'],
+	type: 'array',
+	description:
+		'(OPTIONAL) A mapping of custom metadata in the `"TAG_NAME" "TAG_VALUE"` format to be applied to the GQL Tags of all Data Transactions created. Must be an even number of string values to determine custom metadata. Can NOT be used in conjunction with --metadata-file',
```

Comment:
I don't like ANY of the usages of the word "metadata" here. It's confusing. It's just Arweave tag data.

Reply:
```suggestion
		'(OPTIONAL) A list of custom Arweave tag name and value pairs in the format `"TAG_NAME" "TAG_VALUE"` that will be applied to all file data transactions created during this invocation. Must be an even number of string values. Can NOT be used in conjunction with --metadata-file',
```